test/assets/TestConstants.scala

```diff
@@ -200,4 +201,60 @@ object TestConstants extends ImplicitDateFormatter {
     val obligationsDataErrorJson = Json.parse(obligationsDataErrorString)
 
   }
+
+  object CalcBreakdown {
+    val calculationDataSuccessModel = CalculationDataModel(incomeTaxYTD = Some(1),
```

Comment:
Not that were focussing on validating data too much but the YTD figure is actually the final tax total estimate we supply back